src/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/AnalyticChristoffel.hpp

```diff
@@ -0,0 +1,85 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <memory>
+#include <pup.h>
+#include <string>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/Gauges.hpp"
+#include "Options/Options.hpp"
+#include "PointwiseFunctions/InitialDataUtilities/InitialData.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+class DataVector;
+namespace gsl {
+template <class T>
+class not_null;
+}  // namespace gsl
+template <size_t Dim>
+class Mesh;
+/// \endcond
+
+namespace GeneralizedHarmonic::gauges {
+/*!
+ * \brief Imposes the analytic gauge condition,
+ * \f$H_a=\Gamma_a^{\mathrm{analytic}}\f$ from an analytic solution or analytic
+ * data.
+ *
+ * \warning Assumes \f$\partial_t \Gamma_a=0\f$ i.e. the solution is static.
```

Comment:
I'm confused; you test this on a gauge wave which is not static

Reply:
The derivative being zero is fine (it's just a test). I'm confused on that H_a seems to vanish. That's not what I get analytically...

Reply:
Updated to specify it can be either static or in harmonic gauge. I think technically any spatially and temporally constant function is okay, but H_a=0 is the only realistically used one that I know of.